bigtable/src/test/java/com/m/examples/bigtable/InstanceAdminExampleTest.java

```
@@ -91,18 +94,22 @@ public void after() {
   }
 
   @Test
-  public void testCreateAndDeleteInstance() throws IOException {
+  public void testCreateAndDeleteInstance() {
```

Usually we let the test throw the exception.